templates/lxc-download.in

```diff
@@ -311,11 +311,15 @@ fi
 # Trap all exit signals
 trap cleanup EXIT HUP INT TERM
 
+# /tmp may be mounted in tmpfs or noexec
+if grep -qw '/tmp' /proc/mounts; then
+    DOWNLOAD_TEMP="${LXC_PATH}"
```

Comment:
DOWNLOAD_TEMP should be initialized as empty before this block, we don't want to rely on uninitialized variables.